### DISCUSSION TITLE
Github student pack benefits issue

### DISCUSSION THREAD

**Discussion by manish-cr** [community/community#109078] [GitHub Education] [answered]
### Hi everyone,

I have made a request for the student developer pack and got approved. It has been 72 hours but I am still not able to access the pack. Appreciate any help on this.

![image](https://github.com/community/community/assets/91329141/eb96187d-b461-4ffc-9c51-a4c5e8dd7601)

![image](https://github.com/community/community/assets/91329141/a838b7d5-dc3e-42a2-94a0-d9e376333697)

**Comment by TWTom041** [comment]
Maybe wait a little more, my friend gets that 4 days later.

**Comment by Newer1107** [comment]
I have the same question, it's been 3 days since I applied for the pack and I still heard nothing from github.

**Comment by educatedforeducation** [comment]
I have the exact same question, submitted to github support as 72 hours have passed and Github Support is radio silent. :(

**Reply by Newer1107** [reply]
the moment I did the comment here, within 10 minutes I got the student dev pack.

**Reply by educatedforeducation** [reply]
You legitimately gave me a heart attack because I thought I got a reply from Github Support (Outlook on my phone yelled at me)

Let's hope that also happens to me :)
Were you sent any emails when you got that?

**Reply by Newer1107** [reply]
Yes, you should be getting something like this:
![Screenshot_2024-02-20-00-46-42-80](https://github.com/community/community/assets/116626403/05c4bf15-40ea-46dd-9d1e-cbe44aad92da)

**Reply by educatedforeducation** [reply]
I'll be on the lookout.
Thank you in the meantime ❤️

**Reply by educatedforeducation** [reply]
@Newer1107 Got it also, thank you so much for helping me and actually telling me about this

**Reply by Newer1107** [reply]
glad to hear that, have a good time coding! I'll see you around.

**Comment by manish-cr** [comment, marked as answer]
Hi guys, I got it on the 4th day. Thanks for all the help.
![image](https://github.com/community/community/assets/91329141/719f5622-925f-4de2-ba87-06287944e737)

**Comment by Marek21SK** [comment]
I'm been waiting 8 days and still nothing

**Reply by SignorTacco** [reply]
Same, 8 days :(

**Comment by kanduru-abhiram** [comment]
Hi @manish-cr, please check whether your Application request is approved and the background color of Approved in green.

![image](https://github.com/community/community/assets/85011280/243cdc64-5fc7-4443-aeac-c36224d65c0b)

And wait till the background color of Approved turns to Violet, once turned, you'll get the access to all the benefits of Student Developer pack.

![image](https://github.com/community/community/assets/85011280/8809cebe-d954-4843-9e2f-4ef38cc5b082)

Please accept this as a solution if helped, thanks.
